tor_core/initialize.py

```diff
@@ -106,10 +106,12 @@ def populate_formatting(config):
     config.audio_formatting = ''
     config.video_formatting = ''
     config.image_formatting = ''
+    config.other_formatting = ''
 
     config.audio_formatting = get_wiki_page('format/audio', config)
     config.video_formatting = get_wiki_page('format/video', config)
     config.image_formatting = get_wiki_page('format/images', config)
+    config.image_formatting = get_wiki_page('format/other', config)
```

Comment:
Should probably be `other_formatting` here :P

Reply:
Nah, the folder above is called `format`, it should be pretty obvious

Reply:
Woops nevermind

Reply:
Great catch!